themes/aframe/layout/partials/blog/image.ejs

```diff
@@ -0,0 +1,5 @@
+<% if (image) { %>
+<a class="post__image borderless-link" href="<%= image.href %>">
+  <img src="/images/blog/<%= image.src %>" title="<%- title %>">
```

Comment:
should use `alt`, not `title` - or both. screen readers read `alt`, not `title`.